Unit tests failing due to use of deprecated `pub` command

**Describe the bug**
```
Run pub run coverage:format_coverage --lcov --in=coverage --out=coverage.lcov --packages=.packages --report-on=lib
[13](https://github.com/atsign-foundation/at_server/runs/6400087718?check_suite_focus=true#step:20:13)
/home/runner/work/_temp/b2611307-aedd-4e21-b0c6-8d12ce1403c2.sh: line 1: pub: command not found
[14](https://github.com/atsign-foundation/at_server/runs/6400087718?check_suite_focus=true#step:20:14)
Error: Process completed with exit code 127.
```

**Expected behavior**
Script should use `dart pub ...`

**Additional context**
The release of dart 2.17 is likely to have caused this to surface.

--- a/.github/workflows/at_server.yaml
+++ b/.github/workflows/at_server.yaml
@@ -108,7 +108,7 @@ jobs:
 
       - name: Convert coverage to LCOV format
         working-directory: ${{ env.secondary-working-directory }}
-        run: pub run coverage:format_coverage --lcov --in=coverage --out=coverage.lcov --packages=.packages --report-on=lib
+        run: dart pub run coverage:format_coverage --lcov --in=coverage --out=coverage.lcov --packages=.packages --report-on=lib
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3.1.0

--- a/.github/workflows/at_server.yaml
+++ b/.github/workflows/at_server.yaml
@@ -106,15 +106,16 @@ jobs:
         working-directory: ${{ env.secondary-working-directory }}
         run: dart test --concurrency=1 --coverage="coverage"
 
-      - name: Convert coverage to LCOV format
-        working-directory: ${{ env.secondary-working-directory }}
-        run: dart pub run coverage:format_coverage --lcov --in=coverage --out=coverage.lcov --packages=.packages --report-on=lib
-
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3.1.0
-        with:
-          token: ${{secrets.CODECOV_TOKEN_AT_SERVER}}
-          file: ${{ env.secondary-working-directory }}/coverage.lcov
+#     Commenting out for now, need to investigate and fix but there are hotter fires burning right now
+#      - name: Convert coverage to LCOV format
+#        working-directory: ${{ env.secondary-working-directory }}
+#        run: dart pub run coverage:format_coverage --lcov --in=coverage --out=coverage.lcov --packages=.packages --report-on=lib
+#
+#      - name: Upload coverage to Codecov
+#        uses: codecov/codecov-action@v3.1.0
+#        with:
+#          token: ${{secrets.CODECOV_TOKEN_AT_SERVER}}
+#          file: ${{ env.secondary-working-directory }}/coverage.lcov
 
   # Runs functional tests on at_secondary.
   # If tests are successful, uploads root server and secondary server binaries for subsequent jobs

